Source/Tools/CodeSizeValidation/Main.cpp

```diff
@@ -39,7 +74,7 @@ class CodeSizeValidation final {
   void CalculateBaseStats(FEXCore::Context::Context* CTX, FEXCore::Core::InternalThreadState* Thread);
 private:
   void ClearStats() {
-    RIPToStats.clear();
+    memset(&CurrentStats, 0, sizeof(CurrentStats));
```

Comment:
Seems like this could just be `CurrentStats = {};`

Reply:
Oop, yep, That was before I changed it away from a std::pair so it was complaining about a default constructor not existing. Changed.